src/commands/servicepack/spec/servicepackCommand.spec.ts

```diff
@@ -0,0 +1,137 @@
+import * as deployModule from '../deploy'
+import { ServicePackCommand } from '../servicePackCommand'
+import { Logger, LogLevel, ServerType, Target } from '@sasjs/utils'
+import * as configUtils from '../../../utils/config'
+import { ReturnCode } from '../../../types/command'
+import { setConstants } from '../../../utils'
+
+const defaultArgs = ['node', 'sasjs']
+const target = new Target({
+  name: 'test',
+  appLoc: '/Public/test/',
+  serverType: ServerType.SasViya,
+  contextName: 'test context'
+})
+const source = 'path/to/json'
+describe('ServicePackCommand', () => {
+  beforeAll(async () => {
+    await setConstants()
+  })
+
+  beforeEach(() => {
+    setupMocks()
+  })
+
+  it('should parse sasjs servicepack deploy command', async () => {
+    const args = [...defaultArgs, 'servicepack', 'deploy', '--source', source]
+
+    const command = new ServicePackCommand(args)
```

Comment:
would be nice to put it in function to reduce code duplication

Reply:
refactored specs, fixed.